### DISCUSSION TITLE
GeoJSON map files within GitHub repositories not rendering properly

### DISCUSSION THREAD

**Discussion by rybesh** [community/community#58244] [Repositories]
### Select Topic Area

Bug

### Body

Since the switch away from Leaflet.js for previewing GeoJSON map files within GitHub repositories, they are no longer rendering properly.

See for example:

https://github.com/periodo/periodo-places/blob/master/gazetteers/syrian-governorates.json

The map is zoomed to the correct area, but none of the geometries are shown. This file rendered just fine with the old Leaflet.js map.

**Comment by kohrVid** [comment]
Does anyone know if a recent attempt was made to "fix" this issue? I added an altered version of the Mercator projection to one of my repos but GitHub's UI completely ignores my labeling. Instead the preview just shows the standard Mercator projection supplied by OSM/TomTom, basically rendering a completely different map. I don't really remember this being an issue when I last checked a few months ago so I can't tell if I'm having the same issue as the OP or if something else has gone wrong since they reported the bug.
